doc/reference_accessor.xml

```diff
@@ -2130,9 +2577,7 @@ GROUP BY gid, field1,field2;
 	  <refnamediv>
 		<refname>ST_PatchN</refname>
 
-		<refpurpose>Return the 1-based Nth geometry (face) if the geometry is a
-			POLYHEDRALSURFACE, POLYHEDRALSURFACEM.
-			Otherwise, return NULL.</refpurpose>
+		<refpurpose>Returns the Nth geometry (face) of a PolyhedralSurface.</refpurpose>
```

Comment:
1-based is lost.

Reply:
With the goal of making the summaries shorter and simpler I removed the note about being 1-based.  It's in the main function page description.  

Is there a reason to leave it in the summary?